### PR TITLE
Release 0.5.1 for bpfman-operator

### PR DESCRIPTION
**Pull request by msherif1234**
cut release 0.5.1 for bpfman-operator

- [x] update Makefile
- [x] update changelogs
- [x] bundle csv

### CODE REVIEW

File: changelogs/CHANGELOG-v0.5.1.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+The v0.5.1 release is a patch release which mainly for bug fixes and enhances the developer build and test process.
```
Comment (Billy99) [inline changelogs/CHANGELOG-v0.5.1.md:1]:
```suggestion
The v0.5.1 release is a patch release which mainly contains bug fixes and enhances the developer build and test process.
```

**Review by Billy99 — COMMENTED** [review]
Nit

**Review by Billy99 — COMMENTED** [review]
/LGTM
I'll let @anfredette take a quick pass at it.

**Review by anfredette — APPROVED** [review]
/LGTM

**Pull request merged**